source/backtraceMetric.ts

```diff
@@ -0,0 +1,226 @@
+import { BacktraceClientOptions } from '.';
+import { uuid } from './utils';
+
+declare const __VERSION__: string;
+
+/**
+ * Function to create instance of BacktraceMetric and ping.
+ */
+export default function backtraceMetric(
+  configuration: BacktraceClientOptions,
+): void {
+  new BacktraceMetric(configuration).ping();
+}
+
+/**
+ * Handles Backtrace Metrics.
+ * TODO: Consider converting to a static class since an instance is only created to call a single function once.
```

Comment:
We store single backtraceClient instance per session/lifecycle - you can't create more than 1 backtrace-client instance. I think it would be better if the constructor will be only available inside the package (since we don't export this function outside the package it's fine enough). I think static class won't give us anything because we still need to track the state of the metrics service, so it's not a perfect solution.

Reply:
Yup, I moved BacktraceMetric to be constructed inside the BacktraceClient constructor. It is not exported anywhere else. I think this is fine solution.
_if_ we do expose some functionality to allow a session to be `ticked`/`pinged`/`persisted` we could maybe refactor, but I think we should avoid exposing any functions, based on your concerns about depreciation.